extensions/interactions/ItemSelectionInput/ItemSelectionInput.js

```diff
@@ -97,3 +98,23 @@ oppia.directive('oppiaShortResponseItemSelectionInput', [
     };
   }
 ]);
+
+oppia.factory('itemSelectionInputRulesService', [function() {
+  return {
+    Equals: function(answer, inputs) {
+      return answer.length == inputs.x.length && answer.every(function(val) {
```

Comment:
I think the best solution to this is to add validation in the editor to prevent duplicates. The interaction currently crashes if there are any duplicates (it displays an error: "[ngRepeat:dupes] Duplicates in a repeater are not allowed."), and I don't think it makes sense conceptually to have duplicate choices in item selection.

Then this would work because none of the sets have duplicates. (Actually, will checking that each set is a subset of the other work when there are duplicates?)